ql/src/java/org/apache/hadoop/hive/ql/parse/ParseUtils.java

```diff
@@ -672,4 +672,50 @@ public static String stripIdentifierQuotes(String val) {
     }
     return val;
   }
+
+  /**
+   * Parse the newly generated SQL statement to get a new AST.
+   */
+  public static ReparseResult parseRewrittenQuery(HiveConf conf, Context ctx,
```

Comment:
first should be command, ctx and conf

Reply:
btw, do you even need conf? rewrittenCtx = new Context(conf); --> ctx.getConf()

Reply:
Done.

Reply:
hm, i don't see any changes

Reply:
Conf removed. Done.